Packages/vcs/Lib/vtk_ui/manager.py

```diff
@@ -78,6 +78,7 @@ def detach(self):
             w.detach()
         if self.window.HasRenderer(self.renderer):
             self.window.RemoveRenderer(self.renderer)
+        self.renderer.RemoveAllViewProps()
```

Comment:
Yeah, I basically only added this line as backup in case I screwed something up elsewhere; the "detach" functions are used throughout my `vtk_ui` package to make elements completely remove themselves from the VTK render stack and make them garbage collectable. I'll go ahead and set renderer to `None` here.